lib/house.rb

```diff
@@ -0,0 +1,50 @@
+class House
+  attr_reader :address, :rooms
+  def initialize(price, address)
+    @price = price
+    @address = address
+    @rooms = []
+    @market_average = 500000
```

Comment:
This is okay, but as a matter of best practice, we try to avoid setting instance variables when we don't need to.  Since this is only being used in one method, your `above_market_average?` method, you can just reference that value directly in that method and then it doesn't need to be exposed here.

Reply:
So hard coding it was the ideal solution?  I put it in a variable so it could be changed in one place and referenced wherever it was needed.

Reply:
Your solution is hardcoded too-- it's a value that's explicitly being set to a single value (as opposed to being passed in via an argument).  

For your use case, there's no reason to expose that information to the entire class when you only need it in one method. You should always choose to give things the smallest scope that meets your needs as a matter of security.